src/persist-client/src/internal/state.rs

```diff
@@ -1671,156 +1657,6 @@ pub(crate) mod tests {
         }
     }
 
-    #[mz_ore::test]
-    fn downgrade_since() {
```

Comment:
Instead of removing this test and the other one, is it worth rewriting them to use critical handle sinces instead of read handle ones? Or is the logic sufficiently tested elsewhere?

Reply:
Wfm!